systemd/manager.go

```diff
@@ -130,10 +130,25 @@ func (m *systemdUnitManager) TriggerStart(name string) {
 	}
 }
 
-// TriggerStop asynchronously starts the unit identified by the given name.
+// Stop stops the unit identified by the given name.
+// This function blocks until the underlying unit is actually stopped.
+func (m *systemdUnitManager) Stop(name string) {
+	retChan := make(chan string)
+	m.stopUnit(name, retChan)
+
+	jobReply := <-retChan
```

Comment:
> jobReply := <-retChan 

the block happens here. this call only returns after systemd has the stop job enqueued.